.changeset/giant-radios-compare.md

```diff
@@ -0,0 +1,18 @@
+---
+"@fuel-connectors/burner-wallet-connector": minor
+"@fuel-connectors/fuel-development-wallet": minor
+"@fuel-connectors/walletconnect-connector": minor
+"@fuel-connectors/solana-connector": minor
+"@fuel-connectors/evm-connector": minor
+"@fuel-connectors/fuelet-wallet": minor
+"@fuel-connectors/fuel-wallet": minor
+"@fuels/connectors": minor
+"@fuel-connectors/bako-safe": minor
+"@fuel-connectors/common": minor
+"@fuels/react": minor
+"docs": minor
+---
+
+- Added documentation package, following the fuels-ts documentation
+- Added tsdoc, typedoc config files for each connectors package
+- Added a basic documentation page
```

Comment:
1- instead of multiple topics inside of a single changeset, use one changeset file for each change
2- be mindful on what package you're applying the changes as it will fill a changelog later on